### PR TITLE
EV-6: Redesign About Page

### PR DESCRIPTION
**Pull request by MelenkoOleksandr**
dev
## GitHub Project

* [EV-6](https://chita.atlassian.net/browse/EV-6)

## Code reviewers

- [ ] [na3alis](https://github.com/na3alis)
- [ ] [PopowychSasha](https://github.com/PopowychSasha)
- [x] [StodolaNazariy](https://github.com/StodolaNazariy)
- [x] [Danyil-man](https://github.com/Danyil-man)

### Second Level Review

- [ ] [andrii-kuzmichov](https://github.com/andrii-kuzmichov)

## Summary of issue

Changed About page design to new one. Added adaptive for mobile and tablet versions.

## Summary of change

Engaged Split Panel Component

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Danyil-man — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]
Молодець))

File: src/components/AboutUs/AboutUs.jsx [hunk 1]
```diff
@@ -1,79 +1,100 @@
...
     </div>
   );
 };
 
-AboutUs.defaultProps = {
-  user: {},
-};
+// AboutUs.defaultProps = {
+//   user: {},
+// };
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: Why you comment it?

File: src/components/AboutUs/AboutUs.jsx [hunk 2]
```diff
@@ -1,79 +1,100 @@
...
 
-AboutUs.propTypes = {
-  user: PropTypes.object,
-};
+// AboutUs.propTypes = {
+//   user: PropTypes.object,
+// };
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: Why you comment it?

File: src/components/AboutUs/AboutUs.jsx [hunk 3]
```diff
@@ -1,79 +1,100 @@
 import React from "react";
 import { connect } from "react-redux";
-import PropTypes from "prop-types";
+// import PropTypes from "prop-types";
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: Why you comment it?

File: src/components/AboutUs/AboutUs.jsx [hunk 4]
```diff
@@ -1,79 +1,100 @@
...
 import { NavLink } from "react-router-dom";
 import aboutImg1 from "../../assets/images/aboutUs/aboutImg-1.png";
 import aboutImg2 from "../../assets/images/aboutUs/aboutImg-2.png";
 import aboutImg3 from "../../assets/images/aboutUs/aboutImg-3.png";
 import aboutImg4 from "../../assets/images/aboutUs/aboutImg-4.png";
-import aboutImg5 from "../../assets/images/aboutUs/aboutImg-5.png";
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: Move to diff file

File: src/components/AboutUs/AboutUs.jsx [hunk 5]
```diff
@@ -1,79 +1,100 @@
...
-  { img: aboutImg1, text: SEARCH_EVENTS },
+const INFO_CARDS = [
+  { img: aboutImg1, text: GET_CONNECTED },
   { img: aboutImg2, text: CREATE_YOUR_EVENT },
-  { img: aboutImg3, text: FIND_EVENTS },
-  { img: aboutImg4, text: GET_CONNECTED },
-  { img: aboutImg5, text: HAVE_FUN_TOGETHER },
+  { img: aboutImg3, text: HAVE_FUN_TOGETHER },
+  { img: aboutImg4, text: NEW_CONTACTS },
 ];
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: the same as above

File: src/components/AboutUs/AboutUs.jsx [hunk 6]
```diff
@@ -1,79 +1,100 @@
...
+                <h2 className="member__name">{name}</h2>
+                <p className="member__description">{description}</p>
+              </div>
+            </div>
+          ))}
+        </div>
       </div>
     </div>
   );
 };
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: Refactor it, reduce extra tag

File: src/components/AboutUs/AboutUs.scss [hunk 7]
```diff
@@ -1,96 +1,385 @@
...
+@import "../../styles/mixins.scss";
 
-.about__container {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  margin-top: 50px;
-}
+.info-text {
+  max-width: 50%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 8]
```diff
@@ -1,96 +1,385 @@
...
 
-.about__info {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  flex-basis: 50%;
-}
+  .info-text__header {
+    font-size: 46px;
+    font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 9]
```diff
@@ -1,96 +1,385 @@
...
+    color: $black;
+  }
 
-.about__info_text {
-  width: 515px;
-  font-weight: 600;
-  margin-bottom: 15px;
+  .info-text__body {
+    font-size: 24px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 10]
```diff
@@ -1,96 +1,385 @@
...
-  &:hover {
-    color: darken($color: $lightThemeAccept, $amount: 15%);
-    border-color: darken($color: $lightThemeAccept, $amount: 15%);
-    background-color: #e6e6e6;
-    text-decoration: none;
+.about {
+  .about-banner {
+    position: relative;
+    background: linear-gradient(0deg, rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)),
+      url("https://images.unsplash.com/photo-1519671482749-fd09be7ccebf?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170");
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 11]
```diff
@@ -1,96 +1,385 @@
...
+    background-position: center;
+    background-size: cover;
+    width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 12]
```diff
@@ -1,96 +1,385 @@
...
+
+    .banner-content {
+      @include flex(column, flex-end, center);
+      position: absolute;
+      bottom: 50px;
+      right: 50px;
+
+      .banner-content__quote {
+        font-family: $quoteFont;
+        font-size: 60px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 13]
```diff
@@ -1,96 +1,385 @@
...
+        color: $white;
+      }
+
+      .btn-events {
+        text-decoration: none;
+        padding: 20px 100px;
+        margin-top: 20px;
+        text-align: center;
+        font-size: 24px;
+        font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 14]
```diff
@@ -1,96 +1,385 @@
...
+        color: $black;
+        background: $white;
+        border-radius: 20px;
+
+        &:hover {
+          box-shadow: 0px 5px 55px $lightPink;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 15]
```diff
@@ -1,96 +1,385 @@
...
+        }
+      }
+    }
+  }
+
+  .about-info {
+    width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 16]
```diff
@@ -1,96 +1,385 @@
...
+      @extend .info-text;
+    }
+
+    .info-images {
+      padding: 0 50px;
+      display: flex;
+      border-radius: 20px;
+      overflow: hidden;
+      height: 400px;
+      width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 17]
```diff
@@ -1,96 +1,385 @@
...
+
+      .info-images__item {
+        position: relative;
+        width: calc(100% / 4);
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: 25%?

File: src/components/AboutUs/AboutUs.scss [hunk 18]
```diff
@@ -1,96 +1,385 @@
...
+
+        img {
+          filter: brightness(40%);
+          width: 100%;
+          height: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 19]
```diff
@@ -1,96 +1,385 @@
...
+
+        &:hover img {
+          filter: brightness(70%);
+        }
+
+        .info-images__text {
+          z-index: 1;
+          position: absolute;
+          top: 50%;
+          left: 50%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 20]
```diff
@@ -1,96 +1,385 @@
...
+          transform: translate(-50%, -50%);
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 21]
```diff
@@ -1,96 +1,385 @@
...
+          text-align: center;
+          color: $white;
+          font-size: 2.5rem;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: try to use px and use constants

File: src/components/AboutUs/AboutUs.scss [hunk 22]
```diff
@@ -1,96 +1,385 @@
...
+        }
+      }
+    }
+
+    .right-container {
+      width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 23]
```diff
@@ -1,96 +1,385 @@
...
+      }
+    }
+  }
+
+  .about-members {
+    @include flex(column, center, center);
+
+    .members-title {
+      font-size: 46px;
+      font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 24]
```diff
@@ -1,96 +1,385 @@
...
+
+    .members-container {
+      @include flex(row, center, center);
+
+      .member {
+        @include flex(column, center, center);
+        text-align: center;
+        padding: 50px;
+        box-sizing: border-box;
+        width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 25]
```diff
@@ -1,96 +1,385 @@
...
+
+        .member__image {
+          @include flex(row, center, center);
+          width: 80%;
+
+          img {
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: do not use tag

File: src/components/AboutUs/AboutUs.scss [hunk 26]
```diff
@@ -1,96 +1,385 @@
...
+            width: 80%;
+            height: 80%;
+            border-radius: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 27]
```diff
@@ -1,96 +1,385 @@
...
+            object-fit: cover;
+          }
+        }
+
+        .member__info {
+          width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 28]
```diff
@@ -1,96 +1,385 @@
...
+          padding: 10px;
+          box-sizing: border-box;
+
+          .member__name {
+            font-size: 36px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 29]
```diff
@@ -1,96 +1,385 @@
...
+            font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 30]
```diff
@@ -1,96 +1,385 @@
...
+            color: $black;
+          }
+
+          .member__description {
+            font-size: 18px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 31]
```diff
@@ -1,96 +1,385 @@
...
-    position: absolute;
-    left: 65px;
-    top: 3px;
-    height: 75px;
-    width: 150px;
-    background-color: $lightThemeAccept;
-    z-index: -1;
+@media only screen and (max-width: $mediaPcScreen) {
+  .info-text {
+    .info-text__body {
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Can use short view

File: src/components/AboutUs/AboutUs.scss [hunk 32]
```diff
@@ -1,96 +1,385 @@
...
+      font-size: 20px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 33]
```diff
@@ -1,96 +1,385 @@
...
-.about__photos {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-wrap: wrap;
-  width: 100%;
+@media only screen and (max-width: $mediaFirstTabletScreen) {
+  .info-text {
+    text-align: center;
+    max-width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 34]
```diff
@@ -1,96 +1,385 @@
...
+    margin-left: 0;
+    padding: 0 50px;
+
+    .info-text__header {
+      font-size: 36px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 35]
```diff
@@ -1,96 +1,385 @@
...
+      font-weight: bold;
+      color: $black;
+    }
+
+    .info-text__body {
+      font-size: 20px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 36]
```diff
@@ -1,96 +1,385 @@
...
+@media only screen and (max-width: $mediaFirstTabletScreen) {
+  .info-text {
+    text-align: center;
+    max-width: 100%;
+    margin-left: 0;
+    padding: 0 50px;
+
+    .info-text__header {
+      font-size: 36px;
+      font-weight: bold;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 37]
```diff
@@ -1,96 +1,385 @@
...
+      color: $black;
+    }
+  }
+
+  .about {
+    .about-banner {
+      height: 70vh;
+
+      .banner-content {
+        .banner-content__quote {
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Can use short view

File: src/components/AboutUs/AboutUs.scss [hunk 38]
```diff
@@ -1,96 +1,385 @@
...
+          font-size: 45px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 39]
```diff
@@ -1,96 +1,385 @@
...
+        }
+
+        .btn-events {
+          padding: 15px 50px;
+          margin-top: 15px;
+          font-size: 20px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 40]
```diff
@@ -1,96 +1,385 @@
...
+    }
+
+    .about-info {
+      padding: 50px 0;
+
+      .info-images {
+        height: 300px;
+
+        .info-images__item {
+          .info-images__text {
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Can use short view

File: src/components/AboutUs/AboutUs.scss [hunk 41]
```diff
@@ -1,96 +1,385 @@
...
+            z-index: 1;
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
+            text-align: center;
+            color: $white;
+            font-size: 24px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 42]
```diff
@@ -1,96 +1,385 @@
...
+      .info-images {
+        height: 300px;
+
+        .info-images__item {
+          .info-images__text {
+            z-index: 1;
+            position: absolute;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%, -50%);
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 43]
```diff
@@ -1,96 +1,385 @@
...
+            text-align: center;
+            color: $white;
+            font-size: 24px;
+          }
+        }
+      }
+
+      .right-container {
+        width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 44]
```diff
@@ -1,96 +1,385 @@
...
+        justify-content: flex-end;
+
+        .info-text__right {
+          margin-right: 0;
+        }
+      }
+    }
+
+    .about-members {
+      .members-title {
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Can use short view

File: src/components/AboutUs/AboutUs.scss [hunk 45]
```diff
@@ -1,96 +1,385 @@
...
+        font-size: 36px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 46]
```diff
@@ -1,96 +1,385 @@
...
+      }
+      .members-container {
+        .member {
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Can use short view

File: src/components/AboutUs/AboutUs.scss [hunk 47]
```diff
@@ -1,96 +1,385 @@
...
+          padding: 20px;
+
+          .member__info {
+            padding: 0px;
+            margin-top: 10px;
+
+            .member__name {
+              font-size: 24px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 48]
```diff
@@ -1,96 +1,385 @@
...
+              margin-bottom: 15px;
+            }
+
+            .member__description {
+              font-size: 16px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 49]
```diff
@@ -1,96 +1,385 @@
...
+            }
+          }
+        }
+      }
+    }
+  }
 }
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Do not abuse nesting

File: src/components/AboutUs/AboutUs.scss [hunk 50]
```diff
@@ -1,96 +1,385 @@
...
-  position: relative;
-  overflow: hidden;
-  width: 30%;
-  height: 150px;
-  margin: 0 10px;
-  margin-bottom: 20px;
+@media only screen and (max-width: $mediaSecondTabletScreen) {
+  .about {
+    .about-banner {
+      .banner-content {
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Can use short view

File: src/components/AboutUs/AboutUs.scss [hunk 51]
```diff
@@ -1,96 +1,385 @@
...
+        width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 52]
```diff
@@ -1,96 +1,385 @@
...
+        text-align: center;
+        align-items: center;
+        right: inherit;
+
+        .banner-content__quote {
+          font-size: 40px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 53]
```diff
@@ -1,96 +1,385 @@
...
-  right: 0px;
-  color: $white;
-  background: $transparentBlackBackground;
-  padding: 5px;
-  width: 150px;
-  user-select: none;
+@media only screen and (max-width: $mediaMobileScreen) {
+  .info-text {
+    .info-text__body {
+      font-size: 16px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 54]
```diff
@@ -1,96 +1,385 @@
...
+    }
+
+    .info-text__header {
+      font-size: 30px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 55]
```diff
@@ -1,96 +1,385 @@
...
+      margin-bottom: 10px;
+    }
+  }
+
+  .about {
+    .about-banner {
+      height: 60vh;
+
+      .banner-content {
+        width: 100%;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 56]
```diff
@@ -1,96 +1,385 @@
...
+        text-align: center;
+        align-items: center;
+        right: inherit;
+
+        .banner-content__quote {
+          font-size: 30px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/AboutUs/AboutUs.scss [hunk 57]
```diff
@@ -1,96 +1,385 @@
...
+        }
+
+        .btn-events {
+          padding: 10px 30px;
+          font-size: 16px;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: use constants

File: src/components/SplitPanels/SplitPanels.jsx [hunk 58]
```diff
@@ -83,7 +83,7 @@ const SplitPanels = () => {
       </div>
       <div
         className={["right-side", collapsedIndex === 1 ? "collapsed" : ""].join(
-          " "
+          " ",
```
Comment (andrii-kuzmichov) [inline src/components/SplitPanels/SplitPanels.jsx]: use template literal

**Review by andrii-kuzmichov — CHANGES_REQUESTED** [review]
Fix all comments

File: src/components/AboutUs/AboutUs.scss [hunk 59]
```diff
@@ -1,96 +1,299 @@
...
+  .btn-events {
+    text-decoration: none;
+    padding: 20px 100px;
+    margin-top: 20px;
+    text-align: center;
+    font-size: $aboutUsBodyText;
+    font-weight: $bold;
+    color: $black;
+    background: $white;
+    border-radius: 20px;
```
Comment (Danyil-man) [inline src/components/AboutUs/AboutUs.scss]: const for border radius

**Review by Danyil-man — DISMISSED** [review]

**Review by PopowychSasha — DISMISSED** [review]

File: src/components/AboutUs/AboutContent.js [hunk 60]
```diff
@@ -0,0 +1,27 @@
+import aboutImg1 from "../../assets/images/aboutUs/aboutImg-1.png";
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutContent.js]: Move this file to constants

File: src/components/AboutUs/AboutContent.js [hunk 61]
```diff
@@ -0,0 +1,27 @@
...
+const {
+  GET_CONNECTED,
+  CREATE_YOUR_EVENT,
+  HAVE_FUN_TOGETHER,
+  NEW_CONTACTS,
+  MEMBER1,
+  MEMBER2,
+  MEMBER3,
+  ...TEXT
+} = constants;
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutContent.js]: change to specific import

File: src/components/AboutUs/AboutUs.jsx [hunk 62]
```diff
@@ -1,84 +1,66 @@
...
-  { img: aboutImg1, text: SEARCH_EVENTS },
-  { img: aboutImg2, text: CREATE_YOUR_EVENT },
-  { img: aboutImg3, text: FIND_EVENTS },
-  { img: aboutImg4, text: GET_CONNECTED },
-  { img: aboutImg5, text: HAVE_FUN_TOGETHER },
-];
-
-const AboutUs = ({ user }) => {
+const AboutUs = () => {
   return (
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: drop return

File: src/components/AboutUs/AboutUs.jsx [hunk 63]
```diff
@@ -1,84 +1,66 @@
...
+          <h1 className="info-text__header">{TEXT_BLOCK2_HEADING}</h1>
+          <p className="info-text__body">{TEXT_BLOCK2_BODY}</p>
+        </div>
+      </div>
 
-const mapStateToProps = state => {
-  return {
-    user: state.user,
-  };
+      <div className="about-members">
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.jsx]: move to diff component

File: src/components/AboutUs/AboutUs.scss [hunk 64]
```diff
@@ -1,96 +1,299 @@
 @import "../../styles/variables.scss";
 @import "../../styles/colors.scss";
+@import "../../styles/fonts.scss";
+@import "../../styles/mixins.scss";
+@import "../../styles/pages/about.scss";
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/AboutUs.scss]: Check it out

File: src/components/AboutUs/InfoSection/InfoSection.jsx [hunk 65]
```diff
@@ -0,0 +1,33 @@
...
+    </div>
+    <div className="info-images">
+      {INFO_CARDS.map(({ img, text }) => (
+        <div key={text} className="info-images__item">
+          <img className="info-images__item-img" src={img} alt={text} />
+          <div className="info-images__text">{text}</div>
+        </div>
+      ))}
+    </div>
+
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/InfoSection/InfoSection.jsx:25]: empty line

File: src/components/AboutUs/Members/Members.jsx [hunk 66]
```diff
@@ -0,0 +1,25 @@
...
+import MEMBERS from "../../../mockup-db/aboutUsMembers";
+import "./Members.scss";
+
+const Members = () => (
+  <div className="about-members">
+    <h3 className="members-title">{MEMBERS_TITLE}</h3>
+    <div className="members-container">
+      {MEMBERS.map(({ name, description, img }) => (
+        <div className="member" key={name}>
+          <div className="member__image-container">
```
Comment (andrii-kuzmichov) [inline src/components/AboutUs/Members/Members.jsx]: extra div

File: src/styles/mixins.scss [hunk 67]
```diff
@@ -9,6 +9,17 @@
...
   font-style: $style;
   font-weight: $weight;
+}
 
-
+@mixin centerAbsolute {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (andrii-kuzmichov) [inline src/styles/mixins.scss]: use constants

**Review by andrii-kuzmichov — APPROVED** [review]

**Pull request merged**